### PR TITLE
[NEXUS-5302] update jetty version to 8.1.7.v20120910

### PR DESCRIPTION
**Pull request by jdillon**
https://issues.sonatype.org/browse/NEXUS-5302

### CODE REVIEW

**Comment by adreghiciu** [comment]
+1

File: nexus/nexus-clients/nexus-restlight-clients/nexus-restlight-test-harness/src/main/java/org/sonatype/nexus/restlight/testharness/NxBasicAuthenticator.java [hunk 1]
```diff
@@ -73,7 +73,7 @@ public Authentication validateRequest(ServletRequest req, ServletResponse res, b
                         UserIdentity user = _loginService.login(username,password);
                         if (user!=null)
                         {
-                            renewSessionOnAuthentication(request,response);
+                            renewSession(request,response);
```
Comment (peterlynch) [inline nexus/nexus-clients/nexus-restlight-clients/nexus-restlight-test-harness/src/main/java/org/sonatype/nexus/restlight/testharness/NxBasicAuthenticator.java:76]: intentional?

Comment (jdillon) [inline nexus/nexus-clients/nexus-restlight-clients/nexus-restlight-test-harness/src/main/java/org/sonatype/nexus/restlight/testharness/NxBasicAuthenticator.java:76]: Yes see https://issues.sonatype.org/browse/NEXUS-5302

**Comment by jdillon** [comment]
Related: https://github.com/sonatype/sisu-jetty8/pull/4

**Comment by nabcos** [comment]
+1

**Pull request merged**